src/Http/Request/Body/MainLanguageValidatingRequestBodyParser.php

```diff
@@ -17,6 +18,14 @@ final class MainLanguageValidatingRequestBodyParser implements RequestBodyParser
         '/address',
     ];
 
+    private const PLACE_TRANSLATABLE_FIELDS = [
+        '/name',
+        '/description',
+        '/address',
+        '/bookingInfo/urlLabel',
+        '/priceInfo/[]/name',
```

Comment:
I'd use JSONPath instead of JSON Pointers here. `/priceInfo/[]/name` is not a valid JSON pointer, because JSON pointers always only point to one specific node (like `/priceInfo/0/name`)

JSONPath on the other hand natively supports wildcards for array values, to target every value in an array.

The paths for these properties would be:
```
$.name
$.description
$.address
$.bookingInfo.urlLabel
$.priceInfo[*].name
```

You can get the data from the JSON using these paths with https://github.com/Galbar/JsonPath-PHP

It's an extra dependency but it feels more logical to use a standard approach here than a workaround to keep using JSON pointers in this class.

Reply:
And unrelated, I think `status.reason` should also require at least a value for the `mainLanguage`? This is now not the case in the Respect rules but I think every translatable field should follow this rule?

Reply:
Indeed, it was a rather naive approach to modify the index and work around this new requirement 😀
The updated code now uses [Galbar/JsonPath-PHP](https://github.com/Galbar/JsonPath-PHP)

Two small tweaks were needed:
- when working with an indexed it is required to iterate over all the elements
- the errors use json pointers instead of a json path

But I think the tweaks are easy to understand and read.